src/main/java/com/cloudbees/jenkins/support/SupportAction.java

```diff
@@ -184,7 +184,7 @@ public boolean isAnonymized() {
     }
 
     @RequirePOST
-    public void doDeleteBundles(StaplerRequest req, StaplerResponse rsp) throws ServletException, IOException {
+    public void doDeleteBundles(StaplerRequest2 req, StaplerResponse2 rsp) throws ServletException, IOException {
```

Comment:
BTW better style is to use `HttpResponse` as a return value, in which case you can get rid of the `rsp` parameter and the `ServletException` throw. I am not sure if there is a clean replacement for `req.submittedForm` though, like a `@SomeAnnotation JSONObject form` parameter.

Reply:
Switch to `HttpResponse`